include/mbedtls/asn1.h

```diff
@@ -219,7 +219,7 @@ int mbedtls_asn1_get_bool( unsigned char **p,
                    int *val );
 
 /**
- * \brief       Retrieve an integer ASN.1 tag and its value.
+ * \brief       Retrieve a +ve integer ASN.1 tag and its value.
```

Comment:
I know `+ve` is common engineering shorthand for positive, but it's not acceptable in documentation of the function. And anyway, what you mean here is 'unsigned integer', not 'positive integer'. 

Please fix!

Reply:
Actualy, it's **non-negative** integer, not unsigned integer. Furthermore the description should say what the function does if it finds a negative integer.

Reply:
I think I misunderstood. So we're obtaining a positive integer from a field that does have a sign, but disregarding the sign. Or rather, it's encoded in 2's complement, which we're disregarding. If that's correct, that's what we should say.

Reply:
Yes I meant to say non-negative integer and thought of writing it as +ve. Following commit explains it better hence this PR can be closed.
https://github.com/ARMmbed/mbedtls-restricted/commit/e444adc75836fd6683ced8589de1b9a549155810

Reply:
No, this function returns an error if it finds an INTEGER field with a negative value.